docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,36 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to a
+large set of arrays, and encapsulates a set of arrays more easily.
+
+The instantiation of a MultiView works exactly like a standard View,
+except that it wraps an array-of-pointers::
+
+  int * myarr[2];
+  RAJA::MultiView< int, RAJA::Layout<2> > MView(myarr, N0, ... Nn);

```

Comment:
This would be more clear if it included something like the example, like
 constexpr int N = 12;
  int a[N] = {0};
  int b[N] = {0};
  int * ab[2] = {a, b};

   RAJA::MultiView<int, RAJA::Layout<2, RAJA::Index_type>> abView(ab, N, N);